gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.2-bin.zip
```

Comment:
Jeg tror også gradlew og gradlew.bat må oppdateres når man bumper denne, se https://blog.nishtahir.com/how-to-properly-update-the-gradle-wrapper/